enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -1603,12 +1606,17 @@ func (c *FirecrackerContainer) remove(ctx context.Context) error {
 
 	var lastErr error
 
-	if err := c.machine.Shutdown(ctx); err != nil {
-		log.CtxErrorf(ctx, "Error shutting down machine: %s", err)
-		lastErr = err
-	}
+	// Note: we don't attempt any kind of clean shutdown here, because at this
+	// point either (a) the VM should be paused and we should have already taken
+	// a snapshot, or (b) we are just calling Remove() to force a cleanup
+	// regardless of VM state (e.g. executor shutdown).
+	//
+	// Also note that firecracker doesn't have a clean shutdown mechanism
+	// currently; machine.Shutdown() in the Go SDK doesn't actually shutdown;
+	// instead it sends Ctrl+Alt+Del which just logs out.
```

Comment:
Ctrl+Alt-Del should trigger a reboot which is handled as a shutdown by firecracker: 
https://github.com/firecracker-microvm/firecracker/blob/main/FAQ.md#how-can-i-gracefully-reboot-the-guest-how-can-i-gracefully-poweroff-the-guest

Reply:
Ah yep - removed this part of the comment.